org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/uriresolver/AbstractClasspathResourceResolver.java

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+* Copyright (c) 2020 IBM Corporation and others.
+* All rights reserved. This program and the accompanying materials
```

Comment:
header needs to be updated to dual license

Reply:
Most files seem to only have the 1 license in the lemminx repo. Should I still update to dual license?

Reply:
You're right, sorry my brains were still on lsp4mp mode.